_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -973,10 +973,10 @@ namespace
 
         if (IsMvcProfile(par.mfx.CodecProfile))
         {
-            mfxExtMVCSeqDesc * extMvc = GetExtBuffer(par);
-            mfxExtCodingOption * extOpt = GetExtBuffer(par);
-            if (extOpt->ViewOutput != MFX_CODINGOPTION_ON) // in case of ViewOutput bitstream should contain one view (not all views)
-                mvcMultiplier = extMvc->NumView ? extMvc->NumView : 1;
+            mfxExtMVCSeqDesc const & extMvc = GetExtBufferRef(par);
+            mfxExtCodingOption const & extOpt = GetExtBufferRef(par);
```

Comment:
extMvc must be present because MvcProfile and extOpt must be always? Please check and add a comment.

Reply:
In MfxVideoParam should contains  mfxExtMVCSeqDesc  any way, so I guess we not need comment here